### PR TITLE
Upgraded Microsoft.Extensions.Logging NuGet packages to 2.0.0

### PR DESCRIPTION
**Pull request by DigiBanks99**
With regards to Issue: https://github.com/serilog/serilog-extensions-logging/issues/115

I have bumped the versions of the `Microsoft.Extensions.Logging` and `Microsoft.Extensions.Logging.Abstractions` libraries to 2.0.0 for .net standard and .net framework 4.61. I've also bumped `Serilog` to 2.8.0.

### CODE REVIEW

**Comment by nblumhardt** [comment]
Thanks @DigiBanks99 !

Re-reading, I think the intent of #115 is to use a single dependency version across all target frameworks; perhaps we should make the break in a new major version, and just drop support (from the current version of the package) for TFMs that _Microsoft.Extensions.Logging_ 2.0 doesn't reach? 🤔

**Comment by DigiBanks99** [comment]
@nblumhardt Hmmm... Yeah, That makes more sense. I'll do it that way then.

**Comment by DigiBanks99** [comment]
The `Microsoft.Extensions.Logging.Abstractions` versions seems to be very inconsistent across the various Serilog projects, as seen [here](https://github.com/search?q=org%3Aserilog+microsoft.extensions.logging.abstractions&type=Code), so I'm not really sure if I should bump it to 2.1.0 at the very least and maybe update the remaining repositories as well?

**Comment by nblumhardt** [comment]
@DigiBanks99 I think aiming for consistency will only make a temporary improvement, since the various projects move forwards at their own cadence - I think making the changes here, first, is a good start :-) :+1:

File: src/Serilog.Extensions.Logging/Serilog.Extensions.Logging.csproj [hunk 1]
```diff
@@ -23,17 +23,25 @@
...
   <ItemGroup Condition=" '$(TargetFramework)' == 'net461' or '$(TargetFramework)' == 'netstandard2.0' ">
     <PackageReference Include="Microsoft.Extensions.Logging" Version="2.0.0" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net45' or '$(TargetFramework)' == 'net46' or '$(TargetFramework)' == 'netstandard1.3' ">
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="1.0.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' or '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.0.0" />
```
[the remark this answers is not in the document]
Comment (DigiBanks99) [reply src/Serilog.Extensions.Logging/Serilog.Extensions.Logging.csproj]: It is. There is a hard reference to `Microsoft.Extensions.Logging.ILogger` and `Microsoft.Extensions.Logging.ProviderAliasAttribute` which are located in the `Microsoft.Extensions.Logging`. It seems that it does not pull it package without it being declared as a package reference.

**Comment by nblumhardt** [comment]
I've had another look at this and think we should release it as a 3.0, along with (I guess) a dependency on the 3.0 MEL packages. Looking at v2 of MEL, there's no separate .NET Framework target anymore, so no point maintaining that here, either.

I've just pushed a release from dev -> master, if that's all successful I think we can get this onto dev and start shaping up the v3 :+1:

**Pull request merged**